apis/config/v1beta1/configuration_types.go

```diff
@@ -301,6 +301,17 @@ type Integrations struct {
 	Frameworks []string `json:"frameworks,omitempty"`
 	// PodOptions defines kueue controller behaviour for pod objects
 	PodOptions *PodIntegrationOptions `json:"podOptions,omitempty"`
+
+	// A list of label keys that should be copied from the job into the workload
```

Comment:
One general recommendation when writing documentation and API comments, is to use active voice https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice

Try to adhere.

Reply:
Ok I'll read and follow the style guide. But here I don't see any good way of rephrasing "should be copied" into active voice...

Reply:
Sounds good, there is the exception `Use passive voice if active voice leads to an awkward construction` :)